flow/raster_cache.cc

```diff
@@ -242,7 +242,7 @@ bool RasterCache::Draw(const Layer* layer,
   entry.used_this_frame = true;
 
   if (entry.image.is_valid()) {
-    entry.image.draw(canvas);
+    entry.image.draw(canvas, paint);
```

Comment:
D'oh!  Good catch. Did it trigger a test failure? If not, then it would make sense to add one.

Reply:
Yes, it failed an engine golden test `EmbedderTest.VerifyB143464703WithSoftwareBackend`. I'm so happy that we have that test 😄